<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PackY</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
    <item>
     <layout class="QVBoxLayout" name="vlayout_left">
      <item>
       <widget class="QLabel" name="label_list_of_tasks">
        <property name="text">
         <string>List of tasks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="table_view_session">
        <property name="styleSheet">
         <string notr="true">QTableView { selection-color: #FFFFFF; selection-background-color:  #696969 }</string>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlayout_task_management">
        <item>
         <widget class="QPushButton" name="push_button_create">
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="push_button_remove">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="push_button_edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlayout_task_running">
        <item>
         <widget class="QPushButton" name="push_button_run_all">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Run all</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="push_button_cancel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="group_box_statistics">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Statistics</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_global_progress">
           <property name="text">
            <string>Global progress</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QProgressBar" name="progress_gar_global_progress">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_task_progress">
           <property name="text">
            <string>Task progress</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QProgressBar" name="progress_bar_task_progress">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="vlayout_right">
      <item>
       <widget class="QGroupBox" name="group_box_file_selection">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>File(s) selection</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLineEdit" name="line_edit_source"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_source">
           <property name="text">
            <string>Source</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="push_button_source">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTreeView" name="tree_view_source"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="group_box_output_properties">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Output properties</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="label_packer">
           <property name="text">
            <string>Packer</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_compression_method">
           <property name="text">
            <string>Compression method</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_compression_level">
           <property name="text">
            <string>Compression level</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_destination">
           <property name="text">
            <string>Destination folder</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QGridLayout" name="glayout_packer">
           <item row="0" column="1">
            <widget class="QRadioButton" name="rbutton_tar">
             <property name="text">
              <string>tar</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">button_group_packer_type</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="rbutton_tbz">
             <property name="text">
              <string>tbz</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">button_group_packer_type</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="rbutton_gz">
             <property name="text">
              <string>gz</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">button_group_packer_type</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QRadioButton" name="rbutton_tgz">
             <property name="text">
              <string>tgz</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">button_group_packer_type</string>
             </attribute>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QRadioButton" name="rbutton_bz2">
             <property name="text">
              <string>bz2</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">button_group_packer_type</string>
             </attribute>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="rbutton_zip">
             <property name="text">
              <string>zip</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">button_group_packer_type</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="rbutton_lzma">
             <property name="text">
              <string>lzma</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">button_group_packer_type</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QRadioButton" name="rbutton_tlz">
             <property name="text">
              <string>tlz</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">button_group_packer_type</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QRadioButton" name="rbutton_xz">
             <property name="text">
              <string>xz</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">button_group_packer_type</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="line_edit_destination"/>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="cbox_compression_method">
           <item>
            <property name="text">
             <string>Store</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Deflate</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Optimal (2x slower)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="cbox_compression_level">
           <item>
            <property name="text">
             <string>Normal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Maximum</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fast</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fastest</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="push_button_destination">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new_session"/>
    <addaction name="separator"/>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_options"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_documentation"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_documentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="action_options">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_new_session">
   <property name="text">
    <string>New session</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="button_group_packer_type"/>
 </buttongroups>
</ui>
